web_sdk/BUILD.gn

```diff
@@ -306,6 +311,14 @@ _compile_platform("flutter_dart2js_kernel_sdk_full_sound") {
   null_environment = true
 }
 
+_compile_platform("flutter_dart2wasm_kernel_sdk_full_sound") {
+  sound_null_safety = true
```

Comment:
Do we still need to spell it out?

Reply:
Yes, insofar as I need to pass it as a parameter to `kernel_worker`